cypress/e2e/24_createOrgPageBounty.cy.ts

```diff
@@ -0,0 +1,81 @@
+describe('Create org bounty from Organization Page', () => {
+  const OrgName = 'Ali SoftDev';
+
+  const org: Cypress.Organization = {
+    loggedInAs: 'alice',
+    name: OrgName,
+    description: 'An organization focused on amazing projects.',
+    website: 'https://amazing.org',
+    github: 'https://github.com/amazing'
+  };
+
+  const bounty: Cypress.Bounty = {
+    organization: OrgName,
+    title: 'Bounty',
+    category: 'Web development',
+    coding_language: ['Typescript', 'Javascript', 'Lightning'],
+    description: 'This is available',
+    amount: '123',
+    tribe: 'Amazing Org Tribe',
+    estimate_session_length: 'Less than 3 hour',
+    estimate_completion_date: '09/09/2024',
+    deliverables: 'We are good to go man',
+    assign: ''
+  };
+
+  const orgPageBounty: Cypress.Bounty = {
+    title: 'Test#2',
+    category: 'Web development',
+    coding_language: ['Typescript', 'Javascript', 'Lightning'],
+    description: 'This is available',
+    amount: '123',
+    tribe: 'Amazing Org Tribe',
+    estimate_session_length: 'Less than 3 hour',
+    estimate_completion_date: '09/09/2024',
+    deliverables: 'We are good to go man',
+    assign: ''
+  };
+
+  beforeEach(() => {
+    cy.login(org.loggedInAs);
+    cy.wait(1000);
+    cy.contains(org.loggedInAs).click();
+    cy.wait(1000);
+    cy.create_org(org);
+    cy.wait(1000);
+  });
+
+  it('should verify  bounty tile displays the organization label is clickable and redirects the user to the organizations bounty page and create a new bounty', () => {
+    cy.create_bounty(bounty);
+    cy.wait(5000);
+
+    cy.contains(bounty.title);
+    cy.wait(5000);
+
+    cy.contains(org.name);
+    cy.wait(1000);
+
+    cy.contains(org.name).invoke('removeAttr', 'target').click({ force: true });
+    cy.wait(5000);
+
+    cy.url().should('include', '/org/bounties');
+    cy.wait(5000);
+
+    cy.contains('Post a Bounty').click();
```

Comment:
@aliraza556 Why add this here when you already have it in the create org bounty function?

Reply:
The `create_orgBounty` also searches for the `Post a Bounty` button.

Reply:
@elraphty,
Just Verify this point: 
![image](https://github.com/stakwork/sphinx-tribes-frontend/assets/87068339/83f437cd-0e93-4781-8150-eef93fc4aec2)

Reply:
@elraphty If you don't want this, then I will remove these lines of code.

Reply:
> @elraphty If you don't want this, then I will remove these lines of code

> @elraphty, Just Verify this point: ![image](https://private-user-images.githubusercontent.com/87068339/314160194-83f437cd-0e93-4781-8150-eef93fc4aec2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NzExMjAsIm5iZiI6MTcxMDg3MDgyMCwicGF0aCI6Ii84NzA2ODMzOS8zMTQxNjAxOTQtODNmNDM3Y2QtMGU5My00NzgxLTgxNTAtZWVmOTNmYzRhZWMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDE3NTM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUwMGY1ZWNhODlhNjVjYTA0ODVkODUyZmE0MzZkY2U0ZDA1ODliNTNiNWQzZjc3NDFjNGE0YjAxY2E3MGY3ZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LOXhOIGmhK9VLXophx4sh4idK7OqUl5s4TYM_FmXQEY)
> 
> ![image](https://private-user-images.githubusercontent.com/87068339/314160830-1a1fe15b-e277-4c17-adc5-b07f14d3a296.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NzExMjAsIm5iZiI6MTcxMDg3MDgyMCwicGF0aCI6Ii84NzA2ODMzOS8zMTQxNjA4MzAtMWExZmUxNWItZTI3Ny00YzE3LWFkYzUtYjA3ZjE0ZDNhMjk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDE3NTM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUwYjdhNTk0ZTUyZmI2NjBjZTFjNDhlYzM5MjE5NzkxN2ExYjhmZTkyNjM0NjM5N2RkNzA5ZjBmN2VkZDA2ZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xo5nPuRFRnR06YraoJatl5UxzSjfFT5mSnOnKU70r60)

Oh! perfect, I see now.